VerifyProfileEmail/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "verify-profile-email",
```

Comment:
on which url is this going to be exposed publicly?

Reply:
In host.json we have ""routePrefix": "public" so the full url will be something like APIM_URL/public/verify-profile-email